Framework/DataObjects/inc/MantidDataObjects/TableColumn.h

```diff
@@ -155,14 +155,32 @@ template <class Type> class TableColumn : public API::Column {
    * is throw. In case of an overflow boost::numeric::positive_overflow or
    * boost::numeric::negative_overflow
    * is throw.
-   * @param i :: The index to an element.
+   * @param value :: The value of the element.
    */
-  double toDouble(size_t i) const override {
+  template <typename T> double convertToDouble(const T &value) const {
     typedef
```

Comment:
we need to keep the code this way because 

``/home/osboxes/work/mantid2/Framework/DataObjects/src/TableColumn.cpp:19:1:   required from here
/usr/include/boost/numeric/conversion/converter_policies.hpp:187:95: error: invalid static_cast from type ‘const Mantid::Kernel::V3D’ to type ‘boost::numeric::raw_converter<boost::numeric::conversion_traits<double, Mantid::Kernel::V3D> >::result_type {aka double}’
``